README.md

```diff
@@ -94,23 +94,28 @@ const reducers = combineReducers({
 });
 
 // using createStore
-const store = createStore(reducer, applyMiddleware(taskMiddleware))
+export default createStore(reducer, applyMiddleware(taskMiddleware))
 
+```
+
+Or if use enhancer:
+```js
 // using enhancers
 const initialState = {}
 const middlewares = [taskMiddleware]
 const enhancers = [
   applyMiddleware(...middlewares)
 ]
 
-const store = createStore(reducer, initialState, compose(...enhancers))
+export default createStore(reducer, initialState, compose(...enhancers));
+
 ```
 
 If you mount kepler.gl reducer in another address instead of `keplerGl`, or the kepler.gl reducer is not
 mounted at root of your state, you will need to specify the path to it when you mount the component
 with the `getState` prop.
 
-#### 2. Mount kepler.gl Component
+### 2. Mount Component
```

Comment:
is it worth mentioning Kepler doesn't support server side rendering? People may run into some issues if they try to render kepler using SSR